WIN-111 Layer IT - Make some more integration tests run on windows (#4235)
metalbear-co/mirrord: changelog.d/+layer-it-win-some-more.internal.md

> Made some more existing layer integration tests run on windows as well

Reviewer: ```suggestion
More of the layer tests now run on Windows as well as Unix in CI.
```

Author: 🙏
